deployments/container/Dockerfile.envoy

```diff
@@ -31,13 +35,13 @@ RUN CGO_ENABLED=0 GOOS=linux go build -v -o registry-server ./cmd/registry-serve
 RUN CGO_ENABLED=0 GOOS=linux go build -v -o authz-server ./cmd/authz-server
 
 # Use an Envoy release image to get envoy in the image.
-FROM envoyproxy/envoy:v1.14.4
+FROM envoyproxy/envoy:v1.16.0
 
 ARG DB_CONFIG=registry
 
 COPY deployments/container/RUN-WITH-ENVOY.sh /RUN-WITH-ENVOY.sh
 COPY deployments/envoy/envoy.yaml /etc/envoy/envoy.yaml
-COPY deployments/envoy/proto.pb /proto.pb
+COPY --from=builder /app/deployments/envoy/proto.pb /proto.pb
```

Comment:
Previously this was copying proto.pb from the local build. This gets it from the container build, which allows containers to be built from otherwise-clean checkouts.